unity-renderer/Assets/DCLServices/PortableExperiences/Analytics/PortableExperiencesAnalyticsService.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+
+namespace DCLServices.PortableExperiences.Analytics
+{
+    public class PortableExperiencesAnalyticsService : IPortableExperiencesAnalyticsService
+    {
+        private readonly IAnalytics analytics;
+        private readonly IUserProfileBridge userProfileBridge;
+
+        public PortableExperiencesAnalyticsService(IAnalytics analytics,
+            IUserProfileBridge userProfileBridge)
+        {
+            this.analytics = analytics;
+            this.userProfileBridge = userProfileBridge;
+        }
+
+        public void Dispose() { }
+
+        public void Initialize() { }
+
+        public void Spawn(string pexId)
+        {
+            analytics.SendAnalytic("spawn_portable_experience", new Dictionary<string, string>
+            {
+                { "wallet_id", userProfileBridge.GetOwn().userId },
```

Comment:
probably it is not changed during gameplay, so I would suggest to replace it with a Lazy property that will be filled on first call and then re-used each next time (also in other 2 occurrence of this code)

Reply:
The performance gain i find is minimal, since the request callstack ends in a profile in cache, so i would prefer to always query the user profile bridge to avoid any kind of inconsistencies.
Anyway i unified the `userId` into an internal property to avoid code duplication.